README.md

```diff
@@ -41,8 +42,25 @@ docker build --no-cache -t nginx.org:my_build .
 ```
 The docker image exposes port 8080 as per the convention for container hosting services.
 
-Local development
------------------
+
+Local development with Docker
+-----------------------------
+Use the [Docker Compose file](docker-compose.yaml) to enable local development with just a
```

Comment:
```suggestion
Use the [Docker Compose file](docker-compose.yaml) to enable local development with a
```

style guidance -- avoid the use of ["weasel words"](https://en.wikipedia.org/wiki/Weasel_word) that don't add meaning